library/src/main/java/com/whatsapp/stringpacks/StringPacksLocaleMetaDataProvider.java

```diff
@@ -0,0 +1,32 @@
+package com.whatsapp.stringpacks;
+
+import java.util.Locale;
+
+public interface StringPacksLocaleMetaDataProvider {
+    /**
+     * Applications can return the appropriate pack file id for the locale,
+     * based on their mapping in `pack_id_mapping` setting
+     * @param locale
+     * @return
```

Comment:
I think it would be best if we can also provide an example in the method comment showing an entry in `pack_id_mapping` and then what to do in here.

for example
```suggestion
     /**
     * Applications can return the appropriate pack file id for the locale,
     * based on their mapping in `pack_id_mapping` setting.
     * This identifies which pack file should the library read from for the specific locale
     * for example,
     * // config.json
     * pack_id_mapping = {
     *   "es-rMX": "es",
     *   "es-rES": "es"
     * }
     * 
     * So, for locales "es-MX" or "es-ES" you should return "es" as the pack Id
     * 
     * @param locale
     * @return packId or null if same as locale's language
```